ntoskrnl/inbv/bootanim.rc

```diff
@@ -15,12 +15,10 @@ IDB_HIBERNATE_BAR   BITMAP "inbv/resources/hibernate.bmp"
 IDB_SHUTDOWN_MSG    BITMAP "inbv/resources/shutdown.bmp"
 IDB_BAR_DEFAULT     BITMAP "inbv/resources/barserver.bmp"
 IDB_LOGO_DEFAULT    BITMAP "inbv/resources/logo.bmp"
-IDB_WKSTA_HEADER    BITMAP "inbv/resources/hwksta.bmp"
-IDB_WKSTA_FOOTER    BITMAP "inbv/resources/fwksta.bmp"
+IDB_WKSTA_HEADER    BITMAP "inbv/resources/header.bmp"
 IDB_BAR_WKSTA       BITMAP "inbv/resources/barwksta.bmp"
 IDB_SERVER_LOGO     BITMAP "inbv/resources/logo.bmp"
-IDB_SERVER_HEADER   BITMAP "inbv/resources/hserver.bmp"
-IDB_SERVER_FOOTER   BITMAP "inbv/resources/fserver.bmp"
```

Comment:
I do assume that the `IDB_SERVER_FOOTER` define should be removed also from some .h file within inbv?

Reply:
Good reminder, I removed the unused defines.
I kept the `IDB_WKSTA_HEADER` and `IDB_SERVER_HEADER` as is in case some future contributor wanted to differentiate between ROS Server and ROS Workstation. When I fixed the logon screensaver, other reviewers didn't like removing the option of differentiating between versions.

Reply:
Thank you for for resolving the header-part. Personally I am no fan of bloat-duplication between Server and Workstation. But I don't think it's worth fighting for a few characters in the defines names here. Resolved therefore.